spec/proj4lookup.spec.ts

```diff
@@ -0,0 +1,32 @@
+var express = require("express");
```

Comment:
require -> import

Reply:
Getting this lovely error, so i'm leaving as is

```ts
> terriajs-server@3.3.1 test /home/nfs/wsl-code/terriajs-server
> ts-node node_modules/.bin/jasmine


/home/nfs/wsl-code/terriajs-server/node_modules/ts-node/src/index.ts:500
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
spec/single-page-routing.spec.ts:4:5 - error TS2451: Cannot redeclare block-scoped variable 'supertest'.

4 var supertest = require("supertest");
      ~~~~~~~~~

  spec/proj4lookup.spec.ts:1:7
    1 const supertest = require("supertest");
            ~~~~~~~~~
    'supertest' was also declared here.
```